src/app/server.js

```diff
@@ -32,14 +32,15 @@ const reqDataLimitOfContentEditor = '50mb'
 const reqDataLimitOfContentUpload = '30mb'
 const ekstepEnv = envHelper.EKSTEP_ENV
 const appId = envHelper.APPID
+const externalWhitelistedDomains = envHelper.SUNBIRD_EXTERNAL_CONTENT_WHITELISTED_DOMAINS
 const defaultTenant = envHelper.DEFAUULT_TENANT
 const portal = this
 const Telemetry = require('sb_telemetry_util')
 const telemetry = new Telemetry()
 const telemtryEventConfig = JSON.parse(fs.readFileSync(path.join(__dirname, 'helpers/telemetryEventConfig.json')))
 const producerId = process.env.sunbird_environment + '.' + process.env.sunbird_instance + '.portal'
 let cassandraCP = envHelper.PORTAL_CASSANDRA_URLS
-const contentServiceLocalUrl = envHelper.content_Service_Local_BaseUrl
+// const contentServiceLocalUrl = envHelper.content_Service_Local_BaseUrl
```

Comment:
Remove unused variable.

Reply:
Changes are done this variable was Added by Anoop I commented it as the hook was not allowing me to commit as it was not used at all